app/views/groups/render_json.rb

```diff
@@ -0,0 +1,45 @@
+module GroupsJson
```

Comment:
I like!

I might name this module GroupView though. + the file name doesn't match the module name, which in Rails is the convention.